src/main/webapp/i18n/de/lti.json

```
@@ -87,6 +87,7 @@
                 "linkedSuccessfully": "Verknüpfung der Übungen erfolgreich",
                 "linkedFailed": "Fehler beim Deep-Linking",
                 "link": "Verlinken",
+                "selectToLink": "Übungen zum Verlinken auswählen",
```

Since it's an error message for the user:

```suggestion
                "selectToLink": "Du musst Übungen zum Verlinken auswählen.",
```

---

It's not only an error but also the button text, thus I figured a shorter error message is fine. Moreover, the alert should never be shown, as the button should not be clickable without a selected exercise